feature-libs/user/profile/components/register/register.component.ts

```diff
@@ -139,10 +142,19 @@ export class RegisterComponent implements OnInit, OnDestroy {
   }
 
   submitForm(): void {
-    if (this.registerForm.valid) {
-      this.registerUser();
-    } else {
+    let isAllValid = true;
```

Comment:
Can we append captcha to the `registerForm` instead of adding extra validation steps?